api/routes.py

```diff
@@ -223,6 +223,13 @@ def acquisition_groups(lane_identifier):
 def feed(lane_identifier):
     return app.manager.opds_feeds.feed(lane_identifier)
 
+@library_route('/crawlable/<list_name>')
```

Comment:
There is little I enjoy less than nitpicking URL designs, but since in the near future collections and libraries will also have crawlable feeds, this URL needs to contain an indication that the client is crawling a list rather than a collection of the same name.

I think either of these two setups would work. I prefer the first one because it opens up the possibility that you might get other sorts of OPDS feeds out of these things.

* `/<library>/crawlable`
* `/<library>/lists/<list_name>/crawlable`
* `/<library>/collections/<collection_name>/crawlable`

* `/<library>/crawlable`
* `/<library>/crawlable/lists/<list_name>`
* `/<library>/crawlable/collections/<collection_name>`